src/libs/oci/src/lib.rs

```diff
@@ -47,6 +47,8 @@ pub struct Spec {
     pub root: Option<Root>,
     #[serde(default, skip_serializing_if = "String::is_empty")]
     pub hostname: String,
+    #[serde(default, skip_serializing_if = "String::is_empty")]
+    pub domainname: String,
```

Comment:
Could you add the `domainname` to https://github.com/kata-containers/kata-containers/blob/main/src/libs/protocols/src/trans.rs as well?